scripts/systNtuple_mib.py

```diff
@@ -8,7 +8,7 @@
 import glob
 import subprocess
 from os.path import basename
-import ROOT
+# import ROOT
```

Comment:
You can delete this, also from the other script

Reply:
Done